src/commcare_cloud/fab/fabfile.py

```diff
@@ -118,7 +118,7 @@ def _setup_path():
 
 def load_env():
     env.ccc_environment = get_environment(env.env_name)
-    env.ccc_environment.release_name = env.release_name
+    env.ccc_environment.release_name = getattr(env, "release_name", "NO_RELEASE")
```

Comment:
isn't it confusing to have a release named "NO_RELEASE"?

Reply:
That name should never be used to refer to a real release. The confusing thing is that there are code paths that require this variable (and others that depend on it) to be set when it is not valid and should not be used.

Prior to https://github.com/dimagi/commcare-cloud/pull/5806 this would have been set to a name (a date string) with the same format as other valid release names, but it would not refer to a real release. In that scenario, code that referenced `env.code_root` or other similar variables would have referenced a release that did not exist.

Reply:
With that context, it makes sense.
I am just thinking if someone would read only this code and get confused.
possibly, `NO_NAME` would be less confusing.
(this is not a blocking feedback)